File: ncDados/dPixConfig.vb
Public Class dPixConfig

    Private _cliente As Integer
    Private _cpf As String
    Private _cnpj As String
    Private _nome As String
    Private _chave As String
    Private _appKey As String
    Private _ClientID As String
    Private _ClientSecret As String
    Private _certPath As String
    Private _certPass As String


    Public Property Cliente() As Integer
        Get
            Return _cliente
        End Get
        Set(ByVal value As Integer)
            _cliente = value
        End Set
    End Property

    Public Property Cpf() As String
        Get
            Return _cpf
        End Get
        Set(ByVal value As String)
            _cpf = value
        End Set
    End Property

    Public Property Cnpj() As String
        Get
            Return _cnpj
        End Get
        Set(ByVal value As String)
            _cnpj = value
        End Set
    End Property

    Public Property Nome() As String
        Get
            Return _nome
        End Get
        Set(ByVal value As String)
            _nome = value
        End Set
    End Property
    Public Property AppKey() As String
        Get
            Return _appKey
        End Get
        Set(ByVal value As String)
            _appKey = value
        End Set
    End Property
    Public Property Chave() As String
        Get
            Return _chave
        End Get
        Set(ByVal value As String)
            _chave = value
        End Set
    End Property

    Public Property ClientID() As String
        Get
            Return _ClientID
        End Get
        Set(ByVal value As String)
            _ClientID = value
        End Set
    End Property
    Public Property ClientSecret() As String
        Get
            Return _ClientSecret
        End Get
        Set(ByVal value As String)
            _ClientSecret = value
        End Set
    End Property

    Public Property CertPath() As String
        Get
            Return _certPath
        End Get
        Set(ByVal value As String)
            _certPath = value
        End Set
    End Property

    Public Property CertPass() As String
        Get
            Return _certPass
        End Get
        Set(ByVal value As String)
            _certPass = value
        End Set
    End Property

End Class

File: Modelos/dPixConfig.cs
using System;

namespace Modelos
{
    public class dPixConfig
    {
        private string _banco;
        private int _cliente;
        private string _cpf;
        private string _cnpj;
        private string _nome;
        private string _chave;
        private string _appKey;
        private string _ClientID;
        private string _ClientSecret;
        private string _certPath;
        private string _certPass;
        private string _email;

        public string Banco
        {
            get => _banco;
            set => _banco = value;
        }

        public int Cliente
        {
            get => _cliente;
            set => _cliente = value;
        }

        public string Cpf
        {
            get => _cpf;
            set => _cpf = value;
        }

        public string Cnpj
        {
            get => _cnpj;
            set => _cnpj = value;
        }

        public string Nome
        {
            get => _nome;
            set => _nome = value;
        }

        public string AppKey
        {
            get => _appKey;
            set => _appKey = value;
        }

        public string Chave
        {
            get => _chave;
            set => _chave = value;
        }

        public string ClientID
        {
            get => _ClientID;
            set => _ClientID = value;
        }

        public string ClientSecret
        {
            get => _ClientSecret;
            set => _ClientSecret = value;
        }

        public string CertPath
        {
            get => _certPath;
            set => _certPath = value;
        }

        public string CertPass
        {
            get => _certPass;
            set => _certPass = value;
        }

        public string Email
        {
            get => _email;
            set => _email = value;
        }
    }

}
